get.sh

```diff
@@ -402,7 +400,7 @@ checkURL() {
 	local filename="$1"
 	if [[ $filename =~ "test-image" ]] && [ $TEST_IMAGES_REQUIRED = false ]; then
 		required=false
-	elif [[ $filename =~ "debug-image" ]] && [ $DEBUG_IMAGES_REQUIRED = false ]; then
+	elif [[ $filename =~ "debug-image" ]] || [[ "$jar_name" =~ "debugimage" ]] && [ $DEBUG_IMAGES_REQUIRED = false ]; then
```

Comment:
@keithc-ca is this ok or do I need another set of brackets around the first 2 conditions? It seems to work either way I tried it.

Reply:
I'm not a fan of using the global variable for communicating the answer; using `return` would be a cleaner solution.
Even without going there, this function could be improved:
```
checkURL() {
	local filename="$1"
	if [[ $filename =~ "test-image" ]]; then
		required=$TEST_IMAGES_REQUIRED
	elif [[ $filename =~ "debug-image" ]] || [[ "$jar_name" =~ "debugimage" ]]; then
		required=$DEBUG_IMAGES_REQUIRED
	else
		required=true
	fi
}
```
This might also be a good place to use `case`.